shcDecoder/src/main/java/ca/bc/gov/shcdecoder/ISHCVerifier.kt

```diff
@@ -0,0 +1,12 @@
+package ca.bc.gov.shcdecoder
+
+import ca.bc.gov.shcdecoder.model.ImmunizationRecord
+
+interface ISHCVerifier {
```

Comment:
`I` prefix for interfaces is a .NET thing, I'd drop it

Reply:
Oh I see you've used this in a few places. Nvm, best to keep the codebase consistent

Reply:
I have done the refactoring as you suggested.